recipe/meta.yaml

```diff
@@ -11,25 +11,22 @@ source:
   sha256: 794d07d5a28781231ac335a1561b8442f8648ca07cd518310aeb45d6f0807ef6
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
+  string: py_pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
```

Comment:
This recipe is does not use protobuf. So this should just be: 
`string: pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}`